WorkunitExec.ecl

```diff
@@ -55,6 +55,75 @@ EXPORT WorkunitExec := MODULE
 
         RETURN myESPURL;
     END;
+    
+    /**
+     * Find workunits by name.
+     *
+     * @param   jobName             The jobname of the workunit to execute
+     *                              as a string; REQUIRED
+     * @param   jobState            The state of the job to execute
+     *                              as a string; OPTIONAL, defaults to
+     *                              compiled; 
+     * @param   espURL              The full URL for accessing the esp process
+     *                              running on the HPCC Systems cluster (this
+     *                              is typically the same URL as used to access
+     *                              ECL Watch); set to an empty string to use
+     *                              the URL of the current esp process;
+     *                              OPTIONAL, defaults to an empty string
+     * @param   clusterName         The name of the cluster in which to look
+     *                              for the compiled workunit with provided
+     *                              jobName;OPTIONAL, defualts to thor cluster
```

Comment:
Minor:

- Missing space after semicolon
- "defualts" misspelled
- 'thor' should have apostrophes